mlflow/cli/scorers.py

```diff
@@ -59,3 +59,79 @@ def list_scorers(experiment_id: str, output: str) -> None:
         # Table output format
         table = [[name] for name in scorer_names]
         click.echo(_create_table(table, headers=["Scorer Name"]))
+
+
+@commands.command("register-llm-judge")
+@click.option(
+    "--name",
+    "-n",
+    type=click.STRING,
+    required=True,
+    help="Name for the judge scorer",
+)
+@click.option(
+    "--instructions",
+    "-i",
+    type=click.STRING,
+    required=True,
+    help=(
+        "Natural language instructions for evaluation. Must contain at least one "
+        "template variable: {{ inputs }}, {{ outputs }}, {{ expectations }}, or {{ trace }}. "
+        "See the make_judge documentation for the interpretation of these variables."
+    ),
+)
+@click.option(
+    "--model",
+    "-m",
+    type=click.STRING,
+    required=False,
+    help=(
+        "Model identifier to use for evaluation (e.g., 'openai:/gpt-4'). "
+        "If not provided, uses the default model."
+    ),
+)
+@click.option(
+    "--experiment-id",
+    "-x",
+    envvar=MLFLOW_EXPERIMENT_ID.name,
+    type=click.STRING,
+    required=True,
+    help="Experiment ID to register the judge in. Can be set via MLFLOW_EXPERIMENT_ID env var.",
+)
+def register_llm_judge(name: str, instructions: str, model: str | None, experiment_id: str) -> None:
+    """
+    Register an LLM judge scorer in the specified experiment.
+
+    This command creates an LLM judge using natural language instructions and registers
+    it in an experiment for use in evaluation workflows.
+
+    Examples:
+
+    \b
+    # Register a basic quality judge
+    mlflow scorers register-llm-judge -n quality_judge \\
+        -i "Evaluate if {{ outputs }} answers {{ inputs }}. Return yes or no." -x 123
+
+    \b
+    # Register a judge with custom model
+    mlflow scorers register-llm-judge -n custom_judge \\
+        -i "Check whether {{ outputs }} is professional and formal. \\
+            Rate pass, fail, or na" -m "openai:/gpt-4" -x 123
+
+    \b
+    # Using environment variable
+    export MLFLOW_EXPERIMENT_ID=123
+    mlflow scorers register-llm-judge -n my_judge -i "Check whether {{ outputs }} contains PII"
+    """
+    from mlflow.exceptions import MlflowException
+    from mlflow.genai.judges import make_judge
+
+    try:
+        judge = make_judge(name=name, instructions=instructions, model=model)
+        registered_judge = judge.register(experiment_id=experiment_id)
+        click.echo(
+            f"Successfully created and registered judge scorer '{registered_judge.name}' "
+            f"in experiment {experiment_id}"
+        )
+    except MlflowException as e:
+        raise click.ClickException(str(e))
```

Comment:
Do we need the try catch? If we get an exception from either of the API calls, it will present it accurately in stderr, right?

Reply:
Yeah, this probably doesn't help here. We do raise ClickExceptions in other commands but I can't figure out the rule.